### PR TITLE
fix: fix size limit by removing unneeded function

### PR DESCRIPTION
**Pull request by VGabriel45**

### CODE REVIEW

File: contracts/Nexus.sol [hunk 1]
```diff
@@ -324,22 +324,6 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
...
-    ///         \x19\x01 ||
-    ///         this.domainSeparator ||
-    ///         hashStruct(BiconomyNexusMessage({ hash: `hash`}))
-    ///      )
-    ///
-    /// @param hash The original hash.
-    ///
-    /// @return The corresponding replay-safe hash.
-    function replaySafeHash(bytes32 hash) public view virtual returns (bytes32) {
-        return _eip712Hash(hash);
```
Comment (Aboudjem) [inline contracts/Nexus.sol:340]: if we won't use this in this case it's better to remove `_eip712Hash` func as well

**Review by livingrockrises — APPROVED** [review]
approved. but I think you should do this once everything else is in dev

File: contracts/Nexus.sol (hunk 1 above)
Comment (VGabriel45) [reply contracts/Nexus.sol:340]: Yes will do that

**Comment by VGabriel45** [comment]
> approved. but I think you should do this once everything else is in dev

Will wait for all remediations then pull dev and merge this

**Pull request merged**